formwidgets/Money.php

```diff
@@ -96,6 +96,10 @@ public function getSaveValue($value)
             return FormField::NO_SAVE_DATA;
         }
 
+        if(empty($value)) {
```

Comment:
```suggestion
        if(is_null($value)) {
```
If we have the null value passed I would suggest using `is_null` instead of empty.

Reply:
If that's OK I will merge the code.

Reply:
Sure, go ahead.